src/mudlet.cpp

```diff
@@ -5078,7 +5114,6 @@ void mudlet::setupPreInstallPackages(const QString& gameUrl)
         {qsl(":/echo.mpackage"),                          {qsl("*")}},
         {qsl(":/deleteOldProfiles.mpackage"),             {qsl("*")}},
         {qsl(":/enable-accessibility.mpackage"), {qsl("*")}},
-        {qsl(":/mpkg.mpackage"),                    {qsl("*")}},
```

Comment:
This looks like the `development` branch you started from might be out of date - this was added (by your PR #7676) in 1fe0f77122fe765ec578b299a89d69cb3dbddbba but this PR must have been started before that went in - you will need to either update (`git fetch --all`) your local repository and then update (`git reset <something>`) your `development` branch to be the same as the upstream one here and then do a  `git merge development` into your copy of this PR before pushing it up to this one OR "update" this PR (not forgetting to then fetch this and reset your local copy to match it afterwards)...

Reply:
Ah, you must have done a merge and wiped out #7676 in fcd4fdb6f8e9bbf215292321aaba18cdd215127f ... you need to put it back I guess.